staging/src/k8s.io/apiserver/pkg/endpoints/openapi/openapi.go

```diff
@@ -30,10 +30,10 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/apiserver/pkg/util/trie"
+	"k8s.io/kube-openapi/pkg/util"
```

Comment:
I has been used in apiserver and kube-openapi. Does that mean I need to move it to k8s.io/util?